code/game/objects/items/devices/megaphone.dm

```diff
@@ -32,14 +32,14 @@
 	log_admin("[key_name(user)] used a megaphone to say: >[message]<")
 
 	if((src.loc == user && !user.is_mob_incapacitated()))
-		var/list/mob/living/carbon/human/human_viewers = viewers(user) // slow but we need it
-		for(var/mob/living/carbon/human/listener in human_viewers)
-			if(isyautja(listener)) //NOPE
+		// get mobs in the range of the user
+		var/list/mob/listeners = viewers(user) // slow but we need it
+		for(var/mob/listener in listeners)
+			if(!ishuman(listener) && !isobserver(listener))
```

Comment:
```suggestion
			if(!ishuman_strict(listener) && !isobserver(listener))
```
otherwise yautja (which are just a species of human) won't get caught

Reply:
wasn't aware of that. Will test it out now thanks

Reply:
00f8d12c99f8419cdc83f9063366f62f9faeab35

also fixed long text being visible by filtering out mobs in a new list, and used humansynth_strict instead of is_human like you recommended, because I noticed synths couldn't hear the broadcast either

Reply:
> 

good catch